source/fundamentals/crud/read-operations/change-streams.txt

```diff
@@ -125,15 +124,18 @@ update operations:
    :start-after: begin aggregationExample
    :end-before: end aggregationExample
 
-When the change stream receives an update change event, the preceding code
-example outputs the following text:
+An update operation on the collection should produce output similar to the
+following text:
```

Comment:
I: [Style Guide on "should"](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#s:~:text=the%20setup%20procedure.-,should,-Use%20should%20only)

S: The text you deleted looks fine to me. Otherwise:

```suggestion
An update operation on the collection produces the following output:
```

Reply:
changed everywhere